.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+node_js:
+  - "stable"
+install: npm install
+
+services:
+  - postgresql
+
+cache:
+  directories:
+    - "node_modules"
+
+env: 
+  - CODECLIMATE_REPO_TOKEN=205b460fb18d22d3db847cb98cc3fbbee316a0624d59024b5e15728eb78ef38e
```

Comment:
is this how this should be done?

Reply:
will rectify it
I don't think i can encrypt the token since I don't have access to the travis account
done